File: Julia_Q5_Palindrome.js
const input = prompt("Input: ");

const isPalindrome = (str) => {
  return str === str.split('').reverse().join('');
};

if (isPalindrome(input)) {
  console.log("This word/phrase is a palindrome!");
} else {
  console.log("This word/phrase isn't a palindrome :(");
}

File: Julia_Q5_Palindrome.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        int flag=0;
        Scanner input = new Scanner(System.in); 
        System.out.print("Input: ");
        String str = input.nextLine();
        for (int i=str.length() - 1; i >= 0; i--){
            for (int j=0; j<str.length(); j++){
                if (str.charAt(i) == str.charAt(j)){
                    flag=1;
                }
                else{
                    flag=0;
                }
            }
        }
        if (flag==1){System.out.println("This word/Phrase is Palindrome!");}
        else{System.out.println("This word/Phrase isn't Palindrome :(");}
    }
}